waspc/packages/wasp-config/src/userApi.ts

```diff
@@ -0,0 +1,278 @@
+/** This module defines the user-facing API for defining a Wasp app.
+ */
+import * as AppSpec from './appSpec.js'
+import { GET_USER_SPEC } from './_private.js'
+
+export class App {
+  #userSpec: UserSpec;
+
+  // NOTE: Using a non-public symbol gives us a pacakge-private property.
```

Comment:
Why is this important?

Reply:
IDE completion, discussed in person and added a comment.